<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>FlowChartStudio</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QToolBox" name="toolBox">
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="elementsPage">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>192</width>
          <height>453</height>
         </rect>
        </property>
        <attribute name="label">
         <string>图形元素</string>
        </attribute>
        <widget class="QToolButton" name="addRectangleButton">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>110</y>
           <width>91</width>
           <height>81</height>
          </rect>
         </property>
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset resource="resources.qrc">
           <normaloff>:/images/images/rectangle.png</normaloff>:/images/images/rectangle.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>100</width>
           <height>100</height>
          </size>
         </property>
        </widget>
        <widget class="QToolButton" name="addDiamondButton">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>10</y>
           <width>91</width>
           <height>81</height>
          </rect>
         </property>
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset resource="resources.qrc">
           <normaloff>:/images/images/diamond.png</normaloff>:/images/images/diamond.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>100</width>
           <height>100</height>
          </size>
         </property>
        </widget>
        <widget class="QToolButton" name="addArrowButton">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>210</y>
           <width>91</width>
           <height>81</height>
          </rect>
         </property>
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset resource="resources.qrc">
           <normaloff>:/images/images/arrow.png</normaloff>:/images/images/arrow.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>100</width>
           <height>100</height>
          </size>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="page_2">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>192</width>
          <height>453</height>
         </rect>
        </property>
        <attribute name="label">
         <string>Page 2</string>
        </attribute>
       </widget>
      </widget>
      <widget class="QGraphicsView" name="graphicsView"/>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="fileMenu">
    <property name="title">
     <string>文件</string>
    </property>
   </widget>
   <widget class="QMenu" name="editMenu">
    <property name="title">
     <string>编辑</string>
    </property>
   </widget>
   <widget class="QMenu" name="nodeMenu">
    <property name="title">
     <string>节点</string>
    </property>
   </widget>
   <widget class="QMenu" name="arrowMenu">
    <property name="title">
     <string>箭头</string>
    </property>
   </widget>
   <widget class="QMenu" name="textMenu">
    <property name="title">
     <string>文本框</string>
    </property>
   </widget>
   <addaction name="fileMenu"/>
   <addaction name="editMenu"/>
   <addaction name="nodeMenu"/>
   <addaction name="arrowMenu"/>
   <addaction name="textMenu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
